src/assets/scripts/templates/product.js.liquid

```diff
@@ -53,18 +53,16 @@ ira.cacheSelectors = function () {
 };
 
 var bvEnableScrollToReviews = function () {
```

Comment:
looks good, but let's do some research on that button. I have a hard time believing they just put a non-functional button on that page. Will you create a ticket for that and pull it into todo.